csharp/IceGrid/Greeter/Server/Program.cs

```diff
@@ -20,14 +20,5 @@
 adapter.activate();
 Console.WriteLine($"{greeterName} is listening...");
 
-// Shut down the communicator the program receives a SIGTERM or SIGINT signal.
-Console.CancelKeyPress += (sender, eventArgs) =>
```

Comment:
This is not used at all when the server registers its Process facet, which is always the case.